packages/models-library/src/models_library/projects_access.py

```diff
@@ -25,8 +25,16 @@ class Config:
         extra = Extra.forbid
 
 
+class PositiveIntWithExclusiveMinimumRemoved(PositiveInt):
+    # As we are trying to match this Pydantic model to a historical json schema "project-v0.0.1" we need to remove this
+    # Pydantic does not support exclusiveMinimum boolean https://github.com/pydantic/pydantic/issues/4108
```

Comment:
also here. I wonder if this not overdoing. since our open api was quite old and manually created, I would wonder if the new one is not more correct than the old one.

Reply:
I can can give it another try, the problem here is that there is incopatibility between Draft4 and Draft7 json schema, where the exclusiveMinimum/exclusiveMaximum in the older version was boolean, and in the new one (which also Pydantic use) is integer. Some tests started to fail and the easiest way was just to remove the exclusiveMinimum from the generated json schema.

Reply:
Are you talking about the diff-compare tests? these tests in here? https://github.com/ITISFoundation/osparc-simcore/blob/db9c2dd994edf6982a7f08d62de9d300275b0d63/packages/models-library/tests/test__models_fit_schemas.py
These were created to ensure that what was generated was the same as the original ones. I think that if you now replaced the original ones, we do not need to keep these tests. and we can probably also remove the original json and be good with them. inputs @pcrespov ?

Reply:
Yes, as discussed with @matusdrobuliak66 , the final goal is indeed to remove ``project-v0.0.1`` and create different flavours of Project model for various interfaces (see https://github.com/ITISFoundation/osparc-simcore/issues/2008) but in the meantime (when the refactoring is not fully achieved) every PR has to be self-consistent. 

Therefore, I find this change as a transition solution until we get all new parts in place.